.github/workflows/ci_release.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       - name: Trim branch name
         id: trim_ref
         run: |
-          echo "branch=$(${${{ github.ref }}:11})" >> $GITHUB_OUTPUT
+          echo "branch=${GITHUB_REF#refs/*/}" >> $GITHUB_OUTPUT
```

Comment:
TBH looks like this variable isn't used at all. Same for `debug` below.

Am I missing something?